linkerd/app/outbound/src/lib.rs

```diff
@@ -48,7 +48,7 @@ const EWMA_DEFAULT_RTT: Duration = Duration::from_millis(30);
 const EWMA_DECAY: Duration = Duration::from_secs(10);
 
 #[derive(Clone, Debug)]
-pub struct Config<A: OrigDstAddr = SysOrigDstAddr> {
+pub struct Config<A: OrigDstAddr = DefaultOrigDstAddr> {
     pub proxy: ProxyConfig<A>,
```

Comment:
Actually, since `app-core` dependens on `transport`, it seems like we _could_ just remove the generics everywhere, get rid of the `OrigDstAddr` trait, and and put everything in inherent methods on `DefaultOrigDstAddr` (which could maybe lose the default).

Reply:
Not quite. We use an alternate implementation when we don't need any original dsts (i.e. admin, tap servers). But I've simplified it.